lib/fs.js

```diff
@@ -161,7 +161,7 @@ let FileReadStream;
 let FileWriteStream;
 
 const isWindows = process.platform === 'win32';
-const isOSX = process.platform === 'darwin';
```

Comment:
why did the OSX have to go?

Reply:
Previously we were checking `(isWindows || isOSX)`. Right now, we are running the javascript traverse only in Linux.